example/reqres/respondOnce.js

```diff
@@ -5,23 +5,21 @@
 var msb = require('../..');
 var j = 0;
 
-msb.Responder.createServer({
+msb.Responder.createEmitter({
   namespace: 'example:topic'
-})
-.use(function(request, response, next) {
+}).on('responder', function(responder) {
```

Comment:
I'd suggest to take out it into a separate function. Just to simplify readability